config/sudoers

```diff
@@ -32,6 +32,8 @@ vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/unmount-usb.sh
 vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/intermediate-scripts/create-cert
 vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/intermediate-scripts/create-cert
 vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/intermediate-scripts/sign-message
+vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/src/mount.sh
+vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/src/unmount.sh
```

Comment:
I _think_ we can also get rid of the vx-ui permissions for these scripts. Worth testing to be sure!

Also a reminder to update sudoers-for-dev as well

Reply:
oh good call on sudoers-dev.

I was thinking of leaving it for vx-ui... are we sure every component is using vx-services for this now?

Reply:
I'm 90% sure we don't need it for vx-ui since kiosk-browser USB mounting/unmounting, though still in use, uses /vx/code/app-scripts/mount-usb.sh etc. (and not /vx/code/vxsuite/libs/usb-drive/src/mount.sh etc.). But agree that it's fine for now to just keep it for both. We can clean up as we continue to consolidate USB logic!